### DISCUSSION TITLE
Testing Surface Pro 7+

### DISCUSSION THREAD

**Discussion by radawson** [linux-surface/linux-surface#773] [General]
I have a number of surface pro 7+ devices and am going to install Ubuntu on them... I see that the touch screen isn't operational on the 7+ yet... What can I do to help out? Testing and/or debugging?

**Comment by qzed** [comment]
See https://github.com/linux-surface/intel-precise-touch/issues/8 and https://github.com/quo/ithc-linux. Still work in progress.
